app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -44,13 +46,15 @@ function ServiceInstanceRow($filter, DataService, rowMethods, $uibModal) {
     row.notifications = rowMethods.getNotifications(row.apiObject, row.state);
     row.displayName = getDisplayName();
     row.description = getDescription();
-    row.instanceBindings = getBindings();
   };
 
   row.getSecretForBinding = function(binding) {
     return binding && _.get(row, ['state', 'secrets', binding.spec.secretName]);
   };
 
+  row.showSecretLink = function(binding) {
+    return statusConditionReady(binding);
```

Comment:
Agree, I'll update.  I originally had the filter logic in that function, factored it out to a filter but should have also made this change.